app/models/Service.class.js

```diff
@@ -1,5 +1,9 @@
 'use strict'
 
+/**
+ * @class GOVUKPayService
+ * @property externalId {string} - The name of the gateway account
```

Comment:
This isn't true... the external id is not the name of the gateway account.

Reply:
whoops 😊